### ISSUE TITLE
Battery alarm

### ISSUE THREAD

**Issue by alethiophile** [koreader/koreader#4105]
* KOReader version:
* Device:

#### Issue
There's no current alarm at low battery; it's easy to be reading along and first realize it needs plugged in when the OS shuts down and screen stops responding to touches.

Would be good to have a popup that appears at some threshold (maybe configurable?) to remind you to plug it in.

#### Steps to reproduce

**Comment by KenMaltby** [comment]
Perhaps also one for low memory with a suggestion to restart KOReader?

**Comment by NiLuJe** [comment]
Needs background polling, which opens up a whole other can of worms , IMHO.

**Comment by KenMaltby** [comment]
Aren't they already polled and updated, for the mini-progress bar? That should be often enough for such warnings, if the warning trigger point is set right.

**Comment by NiLuJe** [comment]
Possibly not if disabled and or the status-bar is hidden, or any number of weird corner-cases I might think of?

**Comment by NiLuJe** [comment]
Making everywhere we currently show battery percentage (whenever we show it) make *something* different to make it slightly more obvious that battery is low sounds like a good, non-intrusive start, though ;).

**Comment by Frenzie** [comment]
Only problem is that 20-30% sounds reasonable as low battery, but on my H2O that has always simply meant dead. You easily get a month going from 90 to 50 but that's the effective 100 to 10 no matter what it says.

It's always been that way.

tl;dr "low" percentage should be a setting.
